sr_robot_launch/scripts/construct_robot_description

```diff
@@ -35,8 +35,15 @@ class SrConstructRobotDescription():
         CONST_ARM_TYPE = rospy.get_param("~arm_type")
         self._side = ''
         self._robot_description_params = {}
-        self._CONST_RA_ARM = {'prefix': 'ra', 'ip_address': '10.8.1.1', 'arm_type': CONST_ARM_TYPE}
-        self._CONST_LA_ARM = {'prefix': 'la', 'ip_address': '10.8.2.1', 'arm_type': CONST_ARM_TYPE}
+        TIMEOUT_WAIT_FOR_PARAMS_IN_SECS = 60.0
+
+        self._right_arm_ip = '10.8.1.1'
+        self._left_arm_ip = '10.8.2.1'
+
+        self._wait_for_arm_ip_params(TIMEOUT_WAIT_FOR_PARAMS_IN_SECS)
```

Comment:
Name `_wait_for_arm_ip_params` implies waiting only but it also has side effect of changing _right_arm_ip and _left_arm_ip which are used for this function call only. Wouldn't it be better to do either:
1. Make function _wait_for_arm_ip_params return only boolean if it timed out or not and then get parameters directly here in constructor or use default values thus making _right_arm_ip and _left_arm_ip local variables.
2. Make function _wait_for_arm_ip_params take 2 additional parameters of default IP values and return 2 results which will be either real parameters or defaults depending on timeout.

Reply:
addressed in https://github.com/shadow-robot/sr_interface/pull/586/commits/df44fa98bba485f979ea12e3ef69c766f53209fa